lib/src/kbasesearchengine/search/ObjectData.java

```diff
@@ -179,6 +185,13 @@ public GUID getGUID() {
         return keyProps;
     }
 
+    /** Get the highlighted fields and snippet of the object generated by elasticsearch, if present.
+     * @return the highlighted fields with corresponding snipped to be highlighted.
+     */
+    public Map<String, ArrayList> getHighlight() { return highlight; }
```

Comment:
Don't leak implementation details outside of the API.

Always have the method body on a separate line.